components/Scrollytelling/Scrollytelling.tsx

```diff
@@ -10,7 +10,7 @@ import { Heading, Text, type HeadingType } from '@/components/Typography';
 import { MutedVideoLoop, VideoButton } from '@/components/Video';
 import { getProcessedImage } from '@/utilities/getProcessedImage';
 import { type MarginType } from '@/utilities/datasource';
-import { config } from '@/utilities/config';
+import { siteConfig } from '@/utilities/siteConfig';
```

Comment:
When I did a clean reinstall of packages, I got a message complaining about config - I think config was a reserved word. Message went away after I rename this to siteConfig 🤷🏼‍♀️